### PR TITLE
[5.3] BugFix missing close button in popular network

### PR DESCRIPTION
**Pull request by blackdevelopa**
**Description**

This PR fixes this [bug report](https://github.com/MetaMask/mobile-planning/issues/307#issue-1275256009)

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Screenshots/Recordings**

_If applicable, add screenshots or recordings to visualize the changes_

**Issue**

Progresses #???

### CODE REVIEW

File: app/components/UI/Swaps/components/InfoModal.js [hunk 1]
```diff
@@ -78,29 +87,25 @@ function InfoModal({
...
-              <IonicIcon name="ios-close" style={styles.closeIcon} size={30} />
-            </TouchableOpacity>
+            <>
+              <Text style={styles.messageLimit}>
+                <Text>{message} </Text>
+                {urlText && (
+                  <Text link onPress={url}>
+                    {urlText}
+                  </Text>
+                )}
```
Comment (gantunesr) [inline app/components/UI/Swaps/components/InfoModal.js:103]: **Nice to have**: I see in some parts of the code that you are using a conditional rendering. I think it's better if we take the approach that @owencraston explains in this [document](https://www.notion.so/Components-that-return-null-for-conditional-rendering-c1273016e4aa482496fcf85b8cdccefb) for these cases.

**Review by gantunesr — COMMENTED** [review]
LGTM

@blackdevelopa does this PR also solves the issues that @wachunei pointed out earlier today?

File: app/components/UI/Swaps/components/InfoModal.js (hunk 1 above)
Comment (blackdevelopa) [reply app/components/UI/Swaps/components/InfoModal.js]: Thanks. Updated.

**Comment by blackdevelopa** [comment]
> LGTM
> 
> @blackdevelopa does this PR also solves the issues that @wachunei pointed out earlier today?

@wachunei pointed out [this](https://consensys.slack.com/archives/C02U025CVU4/p1655579852271789?thread_ts=1655510045.020029&cid=C02U025CVU4) and [this](https://consensys.slack.com/archives/C02U025CVU4/p1655741470317989). The first might need to go through [design](https://www.figma.com/file/wfOwLmE52OKY0bSrsQwHPL/Add-popular-custom-network?node-id=1697%3A18293). I have a fix for the second. Raising a PR in a bit.

**Review by gantunesr — APPROVED** [review]
LGTM

**Comment by blackdevelopa** [comment]
> @blackdevelopa the close button is misaligned. See [android](https://www.screencast.com/t/ty6RFtRSGR) & [iOS](https://www.screencast.com/t/WgcoldChM)

Great catch. I have updated this. Please check @cortisiko

**Pull request merged**